### PR TITLE
Address PROTOTYPE comments

### PR DESCRIPTION
**Pull request by AlekseyTs**
Test plan https://github.com/dotnet/roslyn/issues/60433

### CODE REVIEW

**Review by jcouv — APPROVED** [review]
LGTM Thanks (iteration 1)

File: docs/compilers/CSharp/Compiler Breaking Changes - DotNet 7.md [hunk 1]
```diff
@@ -1,5 +1,24 @@
 # This document lists known breaking changes in Roslyn after .NET 6 all the way to .NET 7.
 
+## Unsigned right shift operator
+
+***Introduced in .NET SDK 6.0.400, Visual Studio 2022 version 17.3.***
+Language added support for an "Unsigned Right Shift" operator (`>>>`).
```
Comment (333fred) [inline docs/compilers/CSharp/Compiler Breaking Changes - DotNet 7.md:6]:
```suggestion
The language added support for an "Unsigned Right Shift" operator (`>>>`).
```

File: docs/compilers/CSharp/Compiler Breaking Changes - DotNet 7.md [hunk 2]
```diff
@@ -1,5 +1,24 @@
...
+This disables an ability to consume methods implementing user-defined "Unsigned Right Shift" operators
```
Comment (333fred) [inline docs/compilers/CSharp/Compiler Breaking Changes - DotNet 7.md:7]:
```suggestion
This disables the ability to consume methods implementing user-defined "Unsigned Right Shift" operators
```

**Review by 333fred — COMMENTED** [review]
Done review pass. A couple of grammar fixes.

**Review by 333fred — APPROVED** [review]

**Pull request merged**